build-system/tasks/e2e/package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@babel/register": "7.0.0",
     "babel-regenerator-runtime": "6.5.0",
-    "chromedriver": "2.45.0",
+    "chromedriver": "74.0.0",
```

Comment:
I like that the version numbers are now synced: https://github.com/giggio/node-chromedriver/commit/52d90138bb4e8e224328da2300c9acbed411c6c4

Reply:
me too, that will make it easy to check at a glance 😀